### ISSUE TITLE
Sum(1/(n**3 - 1), (n, -oo, -2)).doit() raise UnboundLocalVariable

### ISSUE THREAD

**Issue by gxyd** [sympy/sympy#9908]
```
>>> n = Symbol('n', integer=True)
>>> Sum(1/(n**3 - 1), (n, -oo, -2)).doit()
---------------------------------------------------------------------------
UnboundLocalError                         Traceback (most recent call last)
<ipython-input-5-fc1048a1af47> in <module>()
----> 1 Sum(1/(n**3 - 1), (n, -oo, -2)).doit()

/home/gxyd/Public/sympy/sympy/concrete/summations.py in doit(self, **hints)
    182                 i = i.label
    183 
--> 184             newf = eval_sum(f, (i, a, b))
    185             if newf is None:
    186                 if f == self.function:

/home/gxyd/Public/sympy/sympy/concrete/summations.py in eval_sum(f, limits)
    587     # this can save time when b-a is big.
    588     # We should try to transform to partial fractions
--> 589     value = eval_sum_symbolic(f.expand(), (i, a, b))
    590     if value is not None:
    591         return value

/home/gxyd/Public/sympy/sympy/concrete/summations.py in eval_sum_symbolic(f, limits)
    687             return r
    688 
--> 689     return eval_sum_hyper(f_orig, (i, a, b))
    690 
    691 

/home/gxyd/Public/sympy/sympy/concrete/summations.py in eval_sum_hyper(f, i_a_b)
    764             if cond == False:
    765                 return None
--> 766         return Piecewise((res1 - res2, cond), (old_sum, True))
    767 
    768     if a == S.NegativeInfinity:

UnboundLocalError: local variable 'res1' referenced before assignment
```

**Comment by leosartaj** [comment]
This looks like an indentation issue to me. L-766 should be indented one more block.

**Comment by leosartaj** [comment]
@irresolute Maybe you can pick this up.

**Comment by rresol** [comment]
@leosartaj  sure!

**Comment by gxyd** [comment]
closing fixed by #9919
